containers/hbadeals/rsem2hbadeals.R

```diff
@@ -30,13 +30,17 @@ if("--help" %in% args | "help" %in% args | (length(args) == 0) | (length(args) =
           
          --help                     - you are reading it
          
-      Optionnal arguments:
+      Optional arguments:
           --sample_colname=chr      - Column name in the metadata file that contains the SRR id, default: 'sample_id'
           --status_colname=chr      - Column name in the metadata file with the control,case status information, default: 'status'
-          --n_cores=value           - Number of cores for hbadeals::hbadeals(), default: 2
+          --n_cores=int             - Number of cores for hbadeals::hbadeals(), default: 2
           --isoform_level=boolean   - TRUE or FALSE. Check ?hbadeals::hbadeals, default: TRUE
-          --mcmc_warmup=value       - Number of warmup iterations for MCMC. Check ?hbadeals::hbadeals for recommended value, default: 20
-          --mcmc_iter=value         - Number of iterations for MCMC. Check ?hbadeals::hbadeals for recommended value, default: 100
+          --mcmc_warmup=int         - Number of warmup iterations for MCMC. Check ?hbadeals::hbadeals for recommended value, default: 20
+          --mcmc_iter=int           - Number of iterations for MCMC. Check ?hbadeals::hbadeals for recommended value, default: 100
+
+          --zeroes_threshold=float  - Fraction expressed as float, default: 0.1
+                                      Example: 0.1 for allowing 10% of observations of the minority class to have zero counts
+                                      in a transcript to not discard it from the countsData table.
```

Comment:
Adds new parameter `zeroes_threshold` as suggested by @karleg and exposes it to the user from the helper script `rsem2hbadeals.R`